pkg/mounter/ossfs.go

```diff
@@ -185,22 +185,24 @@ func (f *fuseOssfs) buildPodSpec(c *FusePodContext, target string) (spec corev1.
 }
 
 func (f *fuseOssfs) AddDefaultMountOptions(options []string) []string {
+	alreadySet := false
+	for _, option := range options {
+		if strings.Contains(option, "dbglevel") {
+			alreadySet = true
+			break
+		}
+	}
+	setDbgLevel := "err"
 	switch dbglevel := f.config.Extra["dbglevel"]; dbglevel {
 	case "":
 	case "debug", "info", "warn", "err", "crit":
-		alreadySet := false
-		for _, option := range options {
-			if strings.Contains(option, "dbglevel") {
-				alreadySet = true
-				break
-			}
-		}
-		if !alreadySet {
-			options = append(options, "dbglevel="+dbglevel)
-		}
+		setDbgLevel = dbglevel
 	default:
 		log.Warnf("invalid ossfs dbglevel: %q", dbglevel)
 	}
+	if !alreadySet {
+		options = append(options, fmt.Sprintf("dbglevel=%s", setDbgLevel))
+	}
```

Comment:
I sugguest this implementation:
```golang
	alreadySet := slices.ContainsFunc(options, func(option string) bool {
		return strings.Contains(option, "dbglevel")
	})
	if !alreadySet {
		switch dbglevel := f.config.Extra["dbglevel"]; dbglevel {
		case "":
		case "debug", "info", "warn", "err", "crit":
			options = append(options, "dbglevel="+dbglevel)
		default:
			log.Warnf("invalid ossfs dbglevel: %q", dbglevel)
		}
	}
```
skip whole switch case if alreadySet. use slices.ContainsFunc to replace the manual loop for clearer semantic.

Reply:
BTW, I think we should verify the dbglevel at `extractFuseContainerConfig`, not here. Maybe refuse to start the CSI driver if it is invalid.